### PR TITLE
Apply black style to test_SeqIO_QualityIO.py and index, version 19.10b0

### PR DESCRIPTION
**Pull request by svalqui**
This pull request addresses issue #2552 

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

This PR applies black style to the below files; about 100 standard black style changes.
test_SeqIO_QualityIO.py
test_SeqIO_index.py

### CODE REVIEW

File: Tests/test_SeqIO_QualityIO.py [hunk 1]
```diff
@@ -376,7 +422,9 @@ def test_qual_negative(self):
         with warnings.catch_warnings():
             warnings.simplefilter("ignore", BiopythonParserWarning)
             self.assertEqual(4, SeqIO.convert(h, "qual", h2, "fastq"))
-        self.assertEqual(h2.getvalue(), """@1117_10_107_F3
+        self.assertEqual(
+            h2.getvalue(),
+            """@1117_10_107_F3
```
Comment (peterjc) [inline Tests/test_SeqIO_QualityIO.py]: Perhaps this is nicer?

```python
        self.assertEqual(
            h2.getvalue(),
            """\
@1117_10_107_F3
...
```

See discussion on #2369

Comment (svalqui) [reply Tests/test_SeqIO_QualityIO.py]: yes, looks better, they harder to spot, amended, Thank you.

File: Tests/test_SeqIO_QualityIO.py [hunk 2]
```diff
@@ -176,46 +194,58 @@ def check_qual_char(self, filename, good_count, count):
...
+    ("tab", 4, 5),
+    ("null", 0, 5),
+]
 for base_name, good_count, full_count in tests:
+
     def funct(name, c1, c2):
-        f = lambda x: x.check_qual_char("Quality/error_qual_%s.fastq" % name, c1, c2)  # noqa: E731
+        f = lambda x: x.check_qual_char(
+            "Quality/error_qual_%s.fastq" % name, c1, c2
+        )  # noqa: E731
```
Comment (peterjc) [inline Tests/test_SeqIO_QualityIO.py]: Flake8 doesn't like the placement of the ``# noqa: E731`` pragma:

```
$ flake8
./Tests/test_SeqIO_QualityIO.py:238:9: E731 do not assign a lambda expression, use a def
```

I would guess it needs to go on the line with the ``lambda`` itself?

Comment (svalqui) [reply Tests/test_SeqIO_QualityIO.py]: Yes, I missed that one out, sorry, I have relocated it. Thank You.

**Comment by peterjc** [comment]
👍

**Pull request merged**